### DISCUSSION TITLE
stopped working fully need help please to set it up again

### DISCUSSION THREAD

**Discussion by vervainnecklace** [sct/overseerr#3661] [Help]
hello how do i fully remove overseerr from my system so i can re install it (my friend helped me install it and then suddenly overseerr told me: you are offline) 
so i cleared cache according to some online recs, then i kept getting a blank error page site couldnt be reached, so i deleted it from docker volume and container
hoping to re-do the process, can someone help? 🫠

**Comment by mduijm** [comment]
better to boot up discord and go to the online support there. 
https://discord.com/invite/overseerr
